@commitlint/prompt/src/library/get-prompt.ts

```diff
@@ -118,7 +121,7 @@ export default function getPrompt(
 		if (enumRule) {
 			const [, [, , enums]] = enumRule;
 
-			enums.forEach((enumerable) => {
+			(enums as string[]).forEach((enumerable) => {
```

Comment:
you should not do typecasts as types should be correct here, you can do something like this
```suggestion
			(Array.isArray(enums) ? enums : enums.values).forEach((enumerable) => {
```